operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx

```diff
@@ -25,19 +25,23 @@ interface IItemProps extends WithStyles<typeof fontStyles> {
   colATitle: string
   colAValue: string
   colBTitle: string
-  colBValue?: string
+  colBValue: string | undefined
 }
 
+type Par = object | string | Array<any> | undefined
+type Ret = string | undefined
+const stringify = (par: Par): Ret => typeof par === 'object' || Array.isArray(par) ? JSON.stringify(par) : par
```

Comment:
Can you add unit test coverage for this

Reply:
replaced with a proper and popular package